app/controllers/item_filters.js

```diff
@@ -54,6 +54,7 @@ export default searchModule.extend({
     clearLocation() {
```

Comment:
Can we change the name to `clearLocationAndRedirect` ?

Reply:
sure @Darshan-Chauhan